utils/preferences/src/extensions/unicode/keywords/collation.rs

```diff
@@ -0,0 +1,11 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+crate::enum_keyword!(Collation {
+    "standard" => Standard,
+    "search" => Search,
+    "phonetic" => Phonetic,
+    "pinyin" => Pinyin,
+    "searchjl" => Searchjl
+}, "co");
```

Comment:
Issue: this list is not comprehensive. Is that intentional?

Reply:
Yes. The intention is to cover the "meaningful" keys that we have values for. If there is a collation key that we do not currently have any code or data for then it is safe to exclude it from those keys and add it once we do.

My idea of evolution of those keys is that we keep growing them as we extend data and logic.

Reply:
Extended to cover https://github.com/unicode-org/cldr/blob/main/common/bcp47/collation.xml and filed https://github.com/unicode-org/icu4x/issues/5018